packages/shared/src/components/labels/TokenLabel.svelte

```diff
@@ -1,22 +1,16 @@
 <script lang="ts">
     import { IToken } from '@core/token'
-    import { FontWeight, Text } from '@ui'
+    import { Text } from '@bloomwalletio/ui'
     import TokenAvatar from '@ui/avatars/TokenAvatar.svelte'
 
     export let token: IToken
 </script>
 
 {#if token}
-    <div class="flex flex-row items-center p-2 space-x-2 text-left bg-gray-100 dark:bg-gray-700 rounded-md">
+    <div class="flex flex-row items-center p-2 space-x-2 text-left bg-surface-1 dark:bg-surface-1-dark rounded-md">
         <TokenAvatar size="sm" {token} />
```

Comment:
For consistency, I think we should remove the size="sm" here. I find it difficult to read the 3 letters.

![image](https://github.com/bloomwalletio/bloom/assets/45079109/6c6acf4e-4e04-4563-b92b-a5763c3bbb42)